### PR TITLE
update the limitations of multi-schema change

### PR DESCRIPTION
**Pull request by tangenta**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Oreoxmt** [comment]
@seiya-annie PTAL

File: mysql-compatibility.md [hunk 1]
```diff
@@ -105,6 +105,7 @@ MySQL 系统变量 `optimizer_switch` 在 TiDB 中是只读的，对查询计划
 TiDB 中，所有支持的 DDL 变更操作都是在线执行的。与 MySQL 相比，TiDB 中的 DDL 存在以下限制：
 
 * 使用 `ALTER TABLE` 语句修改一个表的多个模式对象（如列、索引）时，不允许在多个更改中指定同一个模式对象。例如，`ALTER TABLE t1 MODIFY COLUMN c1 INT, DROP COLUMN c1` 在两个更改中都指定了 `c1` 列，执行该语句会输出 `Unsupported operate same column/index` 的错误。
+* 不支持使用单个 `ALTER TABLE` 语句同时修改多个 TiDB 特有的模式对象，包括 `TIFLASH REPLICA`, `SHARD_ROW_ID_BITS`, `AUTO_ID_CACHE` 等。
```
Comment (Oreoxmt) [inline mysql-compatibility.md:108]:
```suggestion
* 不支持使用单个 `ALTER TABLE` 语句同时修改多个 TiDB 特有的模式对象，包括 `TIFLASH REPLICA`，`SHARD_ROW_ID_BITS`，`AUTO_ID_CACHE` 等。
```

File: sql-statements/sql-statement-alter-table.md [hunk 2]
```diff
@@ -154,7 +154,10 @@ Query OK, 0 rows affected, 1 warning (0.25 sec)
 
 TiDB 中的 `ALTER TABLE` 语法主要存在以下限制：
 
-- 使用 `ALTER TABLE` 语句修改一个表的多个模式对象（如列、索引）时，不允许在多个更改中指定同一个模式对象。
+- 使用 `ALTER TABLE` 语句修改一个表的多个模式对象（如列、索引）时，
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-alter-table.md:157]:
```suggestion
- 使用 `ALTER TABLE` 语句修改一个表的多个模式对象（如列、索引）时：
```

File: sql-statements/sql-statement-alter-table.md [hunk 3]
```diff
@@ -154,7 +154,10 @@ Query OK, 0 rows affected, 1 warning (0.25 sec)
...
+    - 不允许在多个更改中指定同一个模式对象。
+    - TiDB 根据**执行前**的表结构检查合法性。例如 `ALTER TABLE ADD INDEX i(b), DROP INDEX i;` 会报错，因为表结构中不存在名字为 "i" 的索引。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-alter-table.md:159]:
```suggestion
    - TiDB 根据**执行前**的表结构检查合法性。例如 `ALTER TABLE ADD INDEX i(b), DROP INDEX i;` 会报错，因为表结构中不存在名字为 `i` 的索引。
```

File: sql-statements/sql-statement-alter-table.md [hunk 4]
```diff
@@ -154,7 +154,10 @@ Query OK, 0 rows affected, 1 warning (0.25 sec)
...
+    - TiDB 的执行顺序是从左往右逐个执行。该行为在个别场景下和 MySQL 不兼容。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-alter-table.md:160]:
```suggestion
    - TiDB 的执行顺序是从左往右逐个执行,该行为在个别场景下和 MySQL 不兼容。
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

File: sql-statements/sql-statement-alter-table.md [hunk 5]
```diff
@@ -154,7 +154,10 @@ Query OK, 0 rows affected, 1 warning (0.25 sec)
 
 TiDB 中的 `ALTER TABLE` 语法主要存在以下限制：
 
-- 使用 `ALTER TABLE` 语句修改一个表的多个模式对象（如列、索引）时，不允许在多个更改中指定同一个模式对象。
+- 使用 `ALTER TABLE` 语句修改一个表的多个模式对象（如列、索引）时：
+    - 不允许在多个更改中指定同一个模式对象。
+    - TiDB 根据**执行前**的表结构检查合法性。例如 `ALTER TABLE ADD INDEX i(b), DROP INDEX i;` 会报错，因为表结构中不存在名字为 `i` 的索引。
+    - TiDB 的执行顺序是从左往右逐个执行,该行为在个别场景下和 MySQL 不兼容。
```
Comment (tangenta) [inline sql-statements/sql-statement-alter-table.md:160]:
```suggestion
    - TiDB 的执行顺序是从左往右逐个执行，该行为在个别场景下和 MySQL 不兼容。
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Comment by Oreoxmt** [comment]
@bb7133 @Defined2014 PTAL

**Review by Defined2014 — APPROVED** [review]

File: sql-statements/sql-statement-alter-table.md [hunk 6]
```diff
@@ -154,7 +154,10 @@ Query OK, 0 rows affected, 1 warning (0.25 sec)
 
 TiDB 中的 `ALTER TABLE` 语法主要存在以下限制：
 
-- 使用 `ALTER TABLE` 语句修改一个表的多个模式对象（如列、索引）时，不允许在多个更改中指定同一个模式对象。
+- 使用 `ALTER TABLE` 语句修改一个表的多个模式对象（如列、索引）时：
+    - 不允许在多个更改中指定同一个模式对象。
+    - TiDB 根据**执行前**的表结构检查合法性。例如 `ALTER TABLE ADD INDEX i(b), DROP INDEX i;` 会报错，因为表结构中不存在名字为 `i` 的索引。
+    - TiDB 的执行顺序是从左往右逐个执行，该行为在个别场景下和 MySQL 不兼容。
```
Comment (TomShawn) [inline sql-statements/sql-statement-alter-table.md:160]:
```suggestion
    - TiDB 的执行顺序是从左往右逐个执行更改，该行为在个别场景下和 MySQL 不兼容。
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**